### PR TITLE
fix gitattributes due to changed files

### PR DESCRIPTION
**Pull request by Tobion**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,5 +1,6 @@
+/.github export-ignore
+/docs export-ignore
```
Comment (Ocramius) [inline .gitattributes]: These should still be exported, IMO

Comment (Tobion) [reply .gitattributes]: sure fine for me. it wasn't exported once before due to /html-docs export-ignore. i guess the /examples should be removed from the gitattributes as well?

Comment (Ocramius) [reply .gitattributes]: Yes please! HTML docs are long gone. Thanks for looking into this!

Comment (Tobion) [reply .gitattributes]: done

**Pull request merged**